pkgdb2/ui/acls.py

```diff
@@ -262,6 +262,9 @@ def package_give_acls(namespace, package):
         namespaces=acls['namespaces'],
     )
     form.pkgname.data = package
+    if str(form.namespace.data) in ['None', '']:
```

Comment:
And the reason is that it doesn't work, I suspect this is something to do with wtforms.

Checking further into it:
``
print form.namespace.data
print type(form.namespace.data)
``
Returns
``
None
<type 'unicode'>
``
That's why the ``if not`` fails :-/